### PR TITLE
[Term Entry] Go Math Functions: .Tan()

### PR DESCRIPTION
**Pull request by 4hbab**
### Description

- Added the Math.Tan() term for go
- Closes issue #2442 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by 4hbab** [comment]
Hi. I had some problems with the merging as i first didn't pull the cos.md file from the main branch.

**Comment by dakshdeepHERE** [comment]
@4hbab can you link the issue this is fixing? Also if you don't want the other file to appear in this PR then simply copy it and delete the file from the branch or reset the commits and stash the other file. Make sure to copy the file then make a separate branch for it and add the file in that branch to push it separately.

File: content/go/concepts/math-functions/terms/cos/cos.md
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/cos/cos.md:1]: Please remove this file from this PR.

File: content/go/concepts/math-functions/terms/tan/tan.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Tags:
+  - 'Math'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Tan()`** function returns the tangent of the given angle (in radians). The `math` library must be imported in order to use this function.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/tan/tan.md:16]:
```suggestion
The **`Tan()`** function returns the tangent of an angle given in radians. The `math` library must be imported in order to use this function.
```

File: content/go/concepts/math-functions/terms/tan/tan.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Syntax
+
+```pseudo
+
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/tan/tan.md:21]:
```suggestion
```
Remove blank line

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@4hbab I dis a first review and had a couple of comments. The main one of which is you need to delete the `cos.md` file from this PR.

**Comment by SSwiniarski** [comment]
@4hbab I see you updated the tan.md file, but you haven't removed the cos.md file.

**Comment by 4hbab** [comment]
I'm sorry. I'm relatively new to PRs and could tell me how I could remove the cos.md file please?

**Comment by SSwiniarski** [comment]
@4hbab if you're working locally, simply delete the file from your repo and commit the change. If you're doing it on GitHub, just navigate to the file, then click on the three dots to the right of the filename. The last option on the list will be "Delete File".

**Comment by 4hbab** [comment]
Thank you. @SSwiniarski I updated the PR and made changes the pass the automated tests.

**Review by SSwiniarski — APPROVED** [review]
@4hbab Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@4hbab @SSwiniarski LGTM

**Comment by SSwiniarski** [comment]
@4hbab Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/go/math-functions/tan

image

**Pull request merged**